cdxev/initialize_sbom.py

```diff
@@ -0,0 +1,114 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import json
+import warnings
+from datetime import datetime
+from typing import Any, Union
+
+from cyclonedx.model import (  # type: ignore
+    ExternalReference,
+    ExternalReferenceType,
+    Tool,
+    XsUri,
+)
+from cyclonedx.model.bom import Bom, BomMetaData  # type: ignore
+from cyclonedx.model.bom_ref import BomRef  # type: ignore
+from cyclonedx.model.component import Component, ComponentType  # type: ignore
+from cyclonedx.model.contact import (  # type: ignore
+    OrganizationalContact,
+    OrganizationalEntity,
+)
+from cyclonedx.model.dependency import Dependency  # type: ignore
+from cyclonedx.output.json import JsonV1Dot6  # type: ignore
+
+from cdxev import pkg
+
+
+def initialize_sbom(
+    software_name: Union[str, None],
+    version: Union[str, None],
+    supplier_software: Union[str, None],
+    supplier_sbom: Union[str, None],
+) -> dict[str, Any]:
+    """
+    Creates an initial SBOM draft to work with, containing the most basic fields.
+
+    param software_name: the name of the software the sbom is for
+    param version: the version of the sbom the software is for
+    param supplier_software: name of the supplier of the software
+    param supplier_sbom: name of the supplier of the SBOM
+
+    returns: initial SBOM for the software
+    """
+    if software_name is None:
+        software_name = "Name of the software described in the SBOM"
+    if version is None:
+        version = "Version of the software"
+    if supplier_sbom is None:
+        supplier_sbom = "The name of the organization supplying the SBOM"
+    if supplier_software is None:
+        supplier_software = "The name of the organization supplying the software"
```

Comment:
This texts should match the `help`-description from above.

Reply:
Done

Changed supplier_sbom to authors, since supplier refers to the same supplier as thwe one for the component, authors makes more sense in this context.